### DISCUSSION TITLE
Reorder Apps

### DISCUSSION THREAD

**Discussion by ropeFullOfHope** [OnionUI/Onion#1389] [Ideas]
When I installed OnionOS I had the Tweaks app at the bottom and the Packet Manager app at the top. I accidentally removed the Tweaks app and after re-adding it, it's now placed at the top. I'd love if there was a way to reorder the apps.

**Comment by XK9274** [comment]
> When I installed OnionOS I had the Tweaks app at the bottom and the Packet Manager app at the top. I accidentally removed the Tweaks app and after re-adding it, it's now placed at the top. I'd love if there was a way to reorder the apps.

Heya, we added this into 4.3.0beta to sort A-Z, Z-A. Currently no way to have a custom sorted list besides these 2 formats but it would be possible to do with the same logic as this tool and a frontend to decide what goes where.

https://github.com/OnionUI/Onion/pull/1300

Available in the beta and will be in release
